### ISSUE TITLE
Globals should not be exported

### ISSUE THREAD

**Issue by TrySound** [tivac/modular-css#284]
I use the latest modular-css-webpack and got this warnings
```
WARNING in ./dev/src/components/styleguide/gLabelsList/gLabelsList.css
Invalid JS identifier "tab-label-color-0", unable to export
 @ ./dev/src/components/styleguide/gLabelsList/gLabelsList.cmt.js 2:0-36
 @ ./~/multi-entry-loader!

WARNING in ./dev/src/components/styleguide/gLabelsList/gLabelsList.css
Invalid JS identifier "tab-label-color-1", unable to export
 @ ./dev/src/components/styleguide/gLabelsList/gLabelsList.cmt.js 2:0-36
 @ ./~/multi-entry-loader!
```
However code contains just this
```css
.item {
    &:global(.tab-label-color-0) {
        background-color: #1CD48A;
    }

    &:global(.tab-label-color-1) {
        background-color: #ED6EBD;
    }
}
```
I think global names shouldn't be exported, they doesn't make sense, just noise.

**Comment by tivac** [comment]
Just to be sure I understand the complete context, what does that CSS get translated too when you remove the nesting?

**Comment by TrySound** [comment]
Oh, sorry. Forgot :) Just simple postcss-nested
```
.item:global(.tab-label-color-0) {
    background-color: #1CD48A;
}

.item:global(.tab-label-color-1) {
    background-color: #ED6EBD;
}
```

**Comment by tivac** [comment]
Ok, figured but wanted to be sure.

I don't think I agree with this. I prefer in my code to have ALL css come from the modular-css output, so I do want `:global()` rules in there. My app code shouldn't have to care whether or not a rule has been scoped.

These warnings are going to be annoying though, I definitely see that. With rollup you can specify an [`onwarn`](https://github.com/rollup/rollup/wiki/JavaScript-API#onwarn) handler that can filter out these warnings, does webpack allow something similar? I'm open to other ideas but I don't think silently dropping them from the output is a good idea.

**Comment by TrySound** [comment]
@tivac Okay. I agree disabling warnings is not the best. But maybe disable global exporting optionally? And in the same way optionally render named exports.

**Comment by TrySound** [comment]
Just in es modules semantics it looks like exporting jQuery because of using window.jQuery in that module file.

**Comment by TrySound** [comment]
And exporting only locals is something like source of truth and isolation.

**Comment by tivac** [comment]
I'm leery of adding too many knobs and levers to control behavior due to the explosion of testing requirements and maintenance cost it can bring. An option is added to disable exporting `:global` is a possibility but not one I'm thrilled about.

**Comment by tivac** [comment]
Closed by #287

**Comment by TrySound** [comment]
Thanks, Pat
